toolkit/tools/imagecustomizerapi/dirconfig.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//
+
+package imagecustomizerapi
```

Comment:
You need to update the `toolkit/tools/imagecustomizer/docs/configuration.md` file.

Reply:
I will update the readme file once all other design/code changes have been made

Reply:
Updated the `toolkit/tools/imagecustomizer/docs/configuration.md` file